Core/src/org/sleuthkit/autopsy/discovery/ui/DomainSummaryViewer.java

```diff
@@ -83,4 +85,31 @@ void addDomain(DomainWrapper domainWrapper) {
             domainListModel.addElement(domainWrapper);
         }
     }
+
+    /**
+     * Get the list of AbstractFiles which are represented by the selected
+     * document preview.
+     *
+     * @return The list of AbstractFiles which are represented by the selected
+     *         document preview.
+     */
+    String getDomainForSelected() {
+        synchronized (this) {
```

Comment:
I had two places I was calling it, apparently I removed one it is still called in the DomainSummaryViewer's ListSelectionListener. I think I was concerned about the contents of the list or the current selection changing while things were happening so I attempted to synchronize all access to the list and listmodel. I will message you regarding what I should be doing regarding synchronization here. 

In the mean time because I am now only calling it in one place I have refactored the code so that this method makes more sense and is easier to read. It is now void sendPopulateEvent(boolean useDomain) and have fixed the comment.